worlds/ladx/LADXR/assembler.py

```diff
@@ -757,7 +757,7 @@ def getLabels(self) -> ItemsView[str, int]:
 
 def const(name: str, value: int) -> None:
     name = name.upper()
-    assert name not in CONST_MAP
+    assert name not in CONST_MAP or CONST_MAP[name] == value

```

Comment:
```suggestion
    assert name not in CONST_MAP or CONST_MAP[name] == value, "I'd recommend  putting a message here, so that you get more than a mere AssertionError on webhost."

```